### ISSUE TITLE
Inventory / item drops?

### ISSUE THREAD

**Issue by Kassicus** [Kassicus/project-dawn#6]

**Comment by metaphysicalcrab** [comment]
I'm thinking for the most part maybe we avoid item drops all together? Like have a player inventory/loadout, astral stash, and store. Perhaps region bosses drop items and the player can choose to update their loadout with them or send the item/replaced items to the astral stash.

Also was thinking of player loadout. 2 weapon slots (2 1h weapons or 1 2h weapon) where 'q' swaps weapon or in the case of a 2h weapon activates an "active ability", 'e' casts some kinda offensive magic, 'r' casts some kinda healing/defensive magic, and 'f' is something misc/grand (maybe like consumables, grenades, ult magic/skill)

Use 'x' as generic interact button, 'c' as character menu/inventory?

**Comment by Kassicus** [comment]
That would work, I figured if we ever wanted to implement craftable items having certain mob types drop crafting components would be interesting, add a simple prox check and like "f to pickup [itemname]"

**Comment by metaphysicalcrab** [comment]
I low key wish something like bootstrap existed for game menus in python lol

**Comment by Kassicus** [comment]
Yeah, I feel that. I have a system in my character-sheet project (I think thats public?) where I do a ton of interaction shit. I had to build out a system for it from scratch. It takes forever to get a good one, but once you have it you are flowing

**Comment by metaphysicalcrab** [comment]
at the very least, I have maximum creative power with the menu rn XD

IT WILL BE BEAUTIFUL
